client/src/api/users.js

```diff
@@ -0,0 +1,23 @@
+export async function getUserById(id) {
+  const response = await fetch(`http://localhost:5000/users/${id}`);
+  const user = await response.json();
+  return user;
+}
+
+export async function getUsers() {
+  const response = await fetch("http://localhost:5000/users");
+  const users = await response.json();
+  return users;
+}
+
+export async function postUser(name) {
+  const response = await fetch(`http://localhost:5000/users/`, {
+    method: "POST",
+    body: JSON.stringify(name),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+  const newName = await response.json();
+  return newName;
+}
```

Comment:
you should use a constant instead, for not screwing it with typos 😉

```suggestion

const DB_JSON = "http://localhost:5000"

export async function getUserById(id) {
  const response = await fetch(`${DB_JSON}/users/${id}`);
  const user = await response.json();
  return user;
}

export async function getUsers() {
  const response = await fetch(`${DB_JSON}/users`);
  const users = await response.json();
  return users;
}

export async function postUser(name) {
  const response = await fetch(`${DB_JSON}/users/`, {
    method: "POST",
    body: JSON.stringify(name),
    headers: {
      "Content-Type": "application/json",
    },
  });
  const newName = await response.json();
  return newName;
}
```

Reply:
No, this should not be saved in a constant. These paths should all be relative

Reply:
alright sorry